src/groups/mqb/mqbc/mqbc_storageutil.cpp

```diff
@@ -2836,13 +2833,9 @@ void StorageUtil::registerQueueReplicaDispatched(
         if (0 == domain) {
             // Failed to obtain a domain object.
 
-            BALL_LOG_ERROR << "Failed to create queue '" << uri
+            BALL_LOG_ERROR << "Failed to create domain for the queue '" << uri
                            << "', queueKey: [" << queueKey
-                           << "], partitionId: [" << partitionId
-                           << "], rc: " << rc_DOMAIN_FAILURE;
-            if (status) {
-                *status = rc_DOMAIN_FAILURE;
-            }
+                           << "], partitionId: [" << partitionId << "].";
```

Comment:
```suggestion
                           << "], partition: [" << partitionId << "].";
```

And in similar places